static/i18n/qqq.json

```diff
@@ -1,27 +1,28 @@
 {
 	"@metadata": {
 		"authors": [
+			"Amire80",
 			"Veloman Yunkan"
 		]
 	},
-	"name": "Current language to which the string is being translated to.",
-	"suggest-full-text-search": "Text appearing in the suggestion list that, when selected, runs a full text search instead of the title search"
-	, "no-such-book": "Error text when the requested book is not found in the library"
-	, "url-not-found" : "Error text about wrong URL for an HTTP 404 error"
-	, "suggest-search" : "Suggest a search when the URL points to a non existing article"
-	, "random-article-failure" :  "Failure of the random article selection procedure"
-	, "invalid-raw-data-type" : "Invalid DATATYPE was used with the /raw endpoint (/raw/<book>/DATATYPE/...); allowed values are 'meta' and 'content'"
-	, "raw-entry-not-found" : "Entry requested via the /raw endpoint was not found"
-	, "400-page-title" : "Title of the 400 error page"
-	, "400-page-heading" : "Heading of the 400 error page"
-	, "404-page-title" : "Title of the 404 error page"
-	, "404-page-heading" : "Heading of the 404 error page"
-	, "500-page-title" : "Title of the 500 error page"
-	, "500-page-heading" : "Heading of the 500 error page"
-	, "fulltext-search-unavailable" : "Title of the error page returned when search is attempted in a book without fulltext search database"
-	, "no-search-results": "Text of the error page returned when search is attempted in a book without fulltext search database"
-	, "library-button-text": "Tooltip of the button leading to the welcome page"
-	, "home-button-text": "Tooltip of the button leading to the main page of a book"
-	, "random-page-button-text": "Tooltip of the button opening a randomly selected page"
-	, "searchbox-tooltip": "Tooltip displayed for the search box"
+	"name": "{{doc-important|Don't write \"English\" in your language!}}\n\n'''Write the name of ''your'' language in its native script.'''\n\nCurrent language to which the string is being translated to.\n\n'''Important:''' Do not use your language’s word for “English”. Use the word that your language uses to refer to itself. If you translate this message to mean “English” in your language, your change will be reverted.",
```

Comment:
Good that this clarification was added!

The Russian translation got this wrong. I fixed it in a new commit added to this PR, but I suspect that the same issue may be present for other translations included in this PR.

Reply:
We should have a look to TW doc. There for sire a way to request reassment of a string translation without modifying the string in English.

Reply:
We may even remove the name. At which moment we want to display to the user the language we are using to speak to him ?

Reply:
I copied `name` from kiwix-desktop. I suspect that we will need it in our language selection UI. We can get rid of `name` if we either

1. switch to 3-letter language codes in our i18n resource filenames or 
2. enhance the language code-to-name translation function `getLanguageSelfName()`
 
https://github.com/kiwix/libkiwix/blob/f90cc39a522b97e8df4acb5692d6c6b56beaf100/src/opds_dumper.cpp#L177-L183

The first option won't work if we want to support language variations (e.g. en_US) in our translations.

Reply:
Names from languages (in their respective languages) should come to 99 percent from libicu (the other one ate hardcoded). Pretty sure it works already that way.

For the monent I'm not sure this is a good to handle local variants of languages... in particular if this bring complexity and new problem to handle.